plugins/Jobs.cpp

```diff
@@ -268,18 +279,33 @@ bool BedrockPlugin_Jobs::processCommand(BedrockNode* node, SQLite& db, BedrockNo
             throw "402 Invalid priority value";
         }
 
-        // Create this new job
-        db.write("INSERT INTO jobs ( created, state, name, nextRun, repeat, data, priority, parentJobID ) "
-                 "VALUES( " +
-                    SCURRENT_TIMESTAMP() + ", " + 
-                    SQ("QUEUED") + ", " + 
-                    SQ(request["name"]) + ", " + 
-                    safeFirstRun + ", " +
-                    SQ(SToUpper(request["repeat"])) + ", " + 
-                    safeData + ", " + 
-                    SQ(priority) + ", " + 
-                    safeParentJobID +
-                 " );");
+        if (mergeIntoID) {
+            // Update the existing job.
+            if(!db.write("UPDATE JOBS SET "
```

Comment:
I feel like, if we're exposing a method to update an existing job, we might as well be able to update any column that makes sense to update. Why not allow updating priority, nextRun, or any other property? (aside from things like `created` which  don't make sense to change).

> My reasoning is that I think that the first time a unique job is created, it's settings should trump all other attempts to create it.

That's not reasoning, it's just an assertion, and "settings" in this context is arbitrary, why is "data" in particular not a "setting" but everything else is?

> Otherwise, we are just doing a lookup for an existing job and updating it, which isn't an upsert.

An upsert is when you either insert a row, or if a row with the same primary key exists, then you update that one instead, right? That's what we're doing, no? Aside from the columns that comprise the primary key, you can update anything you want in a typical upsert, right?